src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -36,8 +37,8 @@
     @XmlElement(required = true)
     private AdaptedContactDetail address;
 
-    //    @XmlElement(required = true)
-    //    private AdaptedAttendance attendance;
+    @XmlElement
+    private AdaptedAttendance attendances;
 
```

Comment:
Naming issues. Attendances make it sound as it there are multiple attendance objects